src/modules/popmotion-controller/popmotion-controller.ts

```diff
@@ -224,12 +264,21 @@ export const popmotionController = (config: PopmotionControllerConfig = {}): Run
         }
       }
 
+      function setDebugBorder(border = '1px solid transparent') {
```

Comment:
On the `parentElement` we could add custom data attributes to indicate that. Could be useful for styling from the outside.

Reply:
done